### PR TITLE
[11.0][MIG] mail_outbound_static: backport from 12.0

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
- this module allows you to configure the from header for a mail server.

### CODE REVIEW

**Comment by yostashiro** [comment]
@YogeshMahera-SerpentCS I suppose we need to preserve the history when backporting, also?

**Comment by YogeshMahera-SerpentCS** [comment]
Hi, @yostashiro preserved commit.

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

File: mail_outbound_static/models/ir_mail_server.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            split_from = message['From'].rsplit(' <', 1)
+            if len(split_from) > 1:
+                email_from = '%s <%s>' % (
+                    split_from[0], mail_server.smtp_from,
+                )
+            else:
+                email_from = mail_server.smtp_from
+
+            message.replace_header('From', email_from)
+            bounce_alias = self.env['ir.config_parameter'].get_param(
```
Comment (gurneyalex) [inline mail_outbound_static/models/ir_mail_server.py]:
```suggestion
            bounce_alias = self.env['ir.config_parameter'].sudo().get_param(
```

from #575

**Review by gurneyalex — CHANGES_REQUESTED** [review]

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**